userspace/libpman/src/ringbuffer_definitions.h

```diff
@@ -36,7 +36,7 @@ struct ring
 struct ring_buffer
 {
 	struct epoll_event *events;
-	struct ring *rings;
+	struct ring **rings;
```

Comment:
This reflects a libbpf internal change https://github.com/libbpf/libbpf/compare/v1.2.2...v1.3.0#diff-55fb7a783836572dc61e85c6c48456fb88fde09582177410696283be17189b93R37